.storybook/main.js

```diff
@@ -17,4 +17,11 @@ module.exports = {
     },
   },
   framework: '@storybook/react',
+  webpackFinal: async (config) => {
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      '~': path.resolve(__dirname, '../app'),
```

Comment:
Nit pick: 
There is a general practise of using `@` instead of `~` for path alias.

Reply:
@samyakshah3008 Has this been considered as well?

Reply:
If we use `@` then we need to change it everywhere in our codebase where we have used `~` 
Shall I do it in this PR? @harshith-venkatesh @iRohitGaur ? Your thoughts?

Reply:
Please consider it as it is application level change

Reply:
@harshith-venkatesh I think it will more of be a change that is made to all the storybook files and will not affect other files, as the path alias in `main.js` file only applies to storybook.

Reply:
What's the conclusion then? @harshith-venkatesh @yesyash ?

Reply:
I feel that changing from `~` to `@` can be done in separate ticket if required by discussing on team call, as the issue this PR fixes is the configuration part in storybook which is currently lagging in the codebase and storybook server is not starting. 

Any thoughts? 😄

Reply:
sounds like a plan to me @samyakshah3008.

Reply:
*issue

Reply:
> True, but that also means all the files of story book in the application might be affected right. Might there be a chance, due to this condition, few components might not work right in story book?

Disagree if the paths are set right there will not be any problem as I have done this before.

Reply:
I guess I got confused with the intent of the ticket and its impact which had led to this discussion.
Thanks for providing more clarity @yesyash @samyakshah3008

Reply:
Sounds great, I have raised a ticket #143 @harshith-venkatesh @yesyash 😄 
Thanks @yesyash for bringing out this discussion!